### DISCUSSION TITLE
Allow for child rows?

### DISCUSSION THREAD

**Discussion by cgunnels** [rappasoft/laravel-livewire-tables#1693] [Q&A]
Does this project allow you to add child rows? Something like this:
![XQmVv](https://github.com/rappasoft/laravel-livewire-tables/assets/63998/bf12588a-68c9-41f8-9b52-5c0e4ce487a3)

**Comment by lrljoe** [comment]
Not easily, you could lazily load a second table as a column, and set it as collapseAlways(), that would work, and should respect the lazy loading and only render when expanded
